### ISSUE TITLE
this._listeners[eventName] is not iterable

### ISSUE THREAD

**Issue by ryder-james** [ChiriVulpes/scryfall-sdk#70]
Running a fairly simple search:
```javascript
const commanders = [];
Scry.Cards.search('is:commander legal:commander game:paper')
	.on('data', card => {
		commanders.push(card);
	})
	.on('end', () => {
		console.log(`found ${commanders.length} results`);
	});
``` 
But getting this error:
```
<path>\node_modules\scryfall-sdk\out\util\EventEmitter.js:70
        for (const listener of this._listeners[eventName])
                                              ^

TypeError: this._listeners[eventName] is not iterable
    at MagicEmitter.emit (<path>\node_modules\scryfall-sdk\out\util\EventEmitter.js:70:47)
    at MagicEmitter.emit (<path>\node_modules\scryfall-sdk\out\util\MagicEmitter.js:51:22)
    at <path>\node_modules\scryfall-sdk\out\api\Cards.js:433:35
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by ChiriVulpes** [comment]
Hmm, I think this is the same thing fixed by the suggested PR #69, I wonder why this wouldn't be covered by the existing tests. I'll take a look tomorrow. Thanks for the report!

**Comment by ChiriVulpes** [comment]
Fixed as of [v4.2.1](https://www.npmjs.com/package/scryfall-sdk/v/4.2.1) via #69
